tests/openwisp2/settings.py

```diff
@@ -61,8 +61,8 @@
         'OPTIONS': {
             'loaders': [
                 'django.template.loaders.filesystem.Loader',
-                'django.template.loaders.app_directories.Loader',
                 'openwisp_utils.loaders.DependencyLoader',
+                'django.template.loaders.app_directories.Loader',
```

Comment:
I think the swap is fine but I am interested on why it was required?

Reply:
Did it as done here https://github.com/openwisp/openwisp-radius/pull/214
Thanks for pointing out. I will update the README for this too

Reply:
Done @atb00ker